lib/inspec/schema/profile_json.rb

```diff
@@ -0,0 +1,58 @@
+require "lib/inspec/schema/primitives"
+
+# These type occur only when running "inspec json <file>".
+
+module Schema
+  module ProfileJson
```

Comment:
namespacing should match pathing.

Reply:
All rubocop disables are gone as of commit https://github.com/inspec/inspec/pull/4330/commits/e7cb80d927b114bcbb0a3627dba84805f2be3300.

However, to get this visual alignment back we could change chefstyle rule Layout/AlignHash to use EnforcedHashRocketStyle: table.

Reply:
Resolved as of https://github.com/inspec/inspec/pull/4330/commits/e7cb80d927b114bcbb0a3627dba84805f2be3300